### PR TITLE
Cleanup resources after services/guestbook e2e tests.

### PR DESCRIPTION
**Pull request by j3ffml**
Also adds -f flag for stop #4343.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
@a-robinson Are you the right reviewer for this?

**Comment by a-robinson** [comment]
Sure, I'll take it.

File: hack/e2e-suite/guestbook.sh [hunk 1]
```diff
@@ -37,8 +37,9 @@ prepare-e2e
 
 GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
```
Comment (a-robinson) [inline hack/e2e-suite/guestbook.sh]: kubectl -f can act on entire directories? I had no idea...

File: hack/e2e-suite/services.sh [hunk 2]
```diff
@@ -59,7 +59,6 @@ function join() {
 svcs_to_clean=()
```
Comment (a-robinson) [inline hack/e2e-suite/services.sh]: Not affected by your change, but for my own understanding, does anything ever actually get added to this?

**Comment by a-robinson** [comment]
Code LGTM, but you'll have to fix the Travis failure.

File: hack/e2e-suite/guestbook.sh (hunk 1 above)
Comment (jlowdermilk) [inline hack/e2e-suite/guestbook.sh]: Yup. It looks for file extensions that could be schemas and attempts to load them as resources (skipping if invalid schema).

File: hack/e2e-suite/services.sh (hunk 2 above)
Comment (jlowdermilk) [inline hack/e2e-suite/services.sh]: All created services get added to it. See [line 74].(https://github.com/jlowdermilk/kubernetes/blob/e2e/hack/e2e-suite/services.sh#L74)

**Comment by jlowdermilk** [comment]
Shippable failure is flake (passed 1.3, timeout on 1.4).

**Comment by a-robinson** [comment]
Ah, shippable... Thanks for fixing travis!

**Pull request merged**